evil-colemak-basics.el

```diff
@@ -133,8 +133,8 @@ rotated; see evil-colemak-basics-rotate-t-f-j."
           "J" 'evil-find-char-to-backward))
        ((eq evil-colemak-basics-char-jump-commands 'evil-snipe)
         ;; XXX https://github.com/hlissner/evil-snipe/issues/46
-        (evil-snipe-def 1 inclusive "t" "T")
-        (evil-snipe-def 1 exclusive "j" "J")
+        (evil-snipe-def 1 'inclusive "t" "T")
+        (evil-snipe-def 1 'exclusive "j" "J")
```

Comment:
it seems that nowadays, `evil-snipe-def` allows to name the custom functions it created under the hood. the issue linked in the comment https://github.com/hlissner/evil-snipe/issues/46 pointed me to https://github.com/hlissner/evil-snipe/pull/67 which adds 2 extra arguments to `evil-snipe-def`.

i guess it would be better to use those instead, using names that won't conflict with pristine `evil-snipe`, e.g.

- `evil-colemak-basics-snipe-t`
- `evil-colemak-basics-snipe-T`
- `evil-colemak-basics-snipe-j`
- `evil-colemak-basics-snipe-J`

… and then binding those commands on the lines below.

what do you think?